tensorboard/plugins/debugger/debugger_plugin_testlib.py

```diff
@@ -39,16 +39,22 @@
 
 class DebuggerPluginTestBase(tf.test.TestCase):
 
+  def __init__(self, *args, **kwargs):
+    super(DebuggerPluginTestBase, self).__init__(*args, **kwargs)
+    self.debugger_plugin_module = None
+
   def setUp(self):
+    super(DebuggerPluginTestBase, self).setUp()
     # Importing the debugger_plugin can sometimes unfortunately produce errors.
     try:
       # pylint: disable=g-import-not-at-top
       from tensorboard.plugins.debugger import debugger_plugin
       from tensorboard.plugins.debugger import debugger_server_lib
       # pylint: enable=g-import-not-at-top
     except Exception as e:  # pylint: disable=broad-except
-      self.skipTest(
+      raise self.skipTest(
```

Comment:
Why raise this exception? I don't see other places where we raise the result from `skipTest`.

Reply:
We use it in one other place. IIUC it's not necessary, since the method itself raises an exception. Adding raise here too is a harmless addition to make the code more readable. It lets the user know that control flow ends when it hits this line.